api/AudioContext.json

```diff
@@ -229,10 +229,10 @@
                 "version_added": "44"
               },
               "safari": {
-                "version_added": null
+                "version_added": false
```

Comment:
This is updating the `latencyHint` entry, not the `baseLatency` hint as the PR description suggests. Has there been a mishap?

Reply:
> This is updating the `latencyHint` entry, not the `baseLatency` hint as the PR description suggests. Has there been a mishap?

Yes — thanks very much for catching that. I see now that `baseLatency` is already marked unsupported for Safari. What instead needs to be change here is the support data for the *latencyHint* and * sampleRate* parameters for the `AudioContext()` constructor.

So to do that, I have amended and pushed an update to this PR branch.

But what I now realize is that we essentially have duplicate data and that these subfeatures can probably just be removed completely. What I mean is, this PR is now essentially a duplicate of  #7040 — because these parameters are actually members of the `AudioContextOptions` dictionary. So, we need the data in two places.

So shall I further update the patch in this PR to have it just completely remove these subfeatures?

Reply:
This gets into the question of whether BCD should represent dictionaries vs. treating them like arguments and return values which @Elchi3 has told me about at times. I don't see an open issue for it though.

@ddbeck what would you suggest we do?

Reply:
I've been opposed to letting dictionaries stand alone as their own features (I think it's actually a bit like a mixin), but I've also lost that fight every time it's come up (see #6265 for an example). I'm indifferent to the actual outcome here, as long as the data says something consistent. If you want to sync up the options features and the dictionary, that's fine. Or if you want to drop the subfeatures and update the dictionary, then that's fine too. Author's choice. 😄

Reply:
+1. I'm still pro inline (i.e. "treating them like arguments and return values") as I mentioned in https://github.com/mdn/browser-compat-data/issues/6265#issuecomment-669278409.

But yes, probably author's choice for now until we all agree and fix this generally? :)

Reply:
I've filed https://github.com/mdn/browser-compat-data/issues/7287 about dictionaries. For this PR, updating them makes sense.